File: estudos/programacao_orientada_a_objetos/java_oop/br/com/remember/oop/Person.java
package br.com.remember.oop; //all classes in the same package to be mutually visible

//class definition: Person
public class Person {

    public  String name; //declaring attribute 1
    private int    age;  //declaring attribute 2

    //constructor function
    public Person (String name, int age) {
        this.name = name; //instantiating attribute 1
        this.age = age;   //instantiating attribute 2
    }

    //method 1
    public void greet () {
        System.out.println("Hello, " + this.name + "!");
    }

    //method 2
    public void howOld () {
        System.out.println(this.name + " is " + this.age + " years old.");
    }
}

File: estudos/programacao_orientada_a_objetos/python_oop/classes/Person.py
#class definition: Person
class Person :

    #constructor function
    def __init__ (self, name, age) :
        self.name = name #attribute 1 (declaration + instantiation)
        self.__age = age #attribute 2 (declaration + instantiation)

    #method 1 
    def greet (self) :
        print("Hello, {}!".format(self.name))
    
    #method 2
    def howOld (self) :
        print("{} is {} years old.".format(self.name, self.__age))